contracts/IbetShare.sol

```diff
@@ -499,6 +551,108 @@ contract IbetShare is Ownable, IbetStandardTokenInterface {
         return true;
     }
 
+    /// @notice 移転申請
+    /// @param _to 移転先アドレス
+    /// @param _value 移転数量
+    /// @param _data イベント出力用の任意のデータ
+    function applyForTransfer(address _to, uint256 _value, string memory _data)
+        public
+    {
+        // <CHK>
+        //  1) 移転時の発行体承諾が不要な場合
+        //  2) 数量が残高を超えている場合
+        //  -> REVERT
+        if (transferApprovalRequired == false ||
+            balanceOf(msg.sender) < _value)
+        {
+            revert();
+        }
+
+        // <CHK>
+        // 個人情報登録済みチェック
+        // 発行体への移転の場合はチェックを行わない
+        if (_to != owner) {
+            require(PersonalInfo(personalInfoAddress).isRegistered(_to, owner) == true);
+        }
+
+        balances[msg.sender] -= _value;
```

Comment:
[qa]`SafeMath`じゃなくても大丈夫ですかね？（他四則演算箇所同様）

Reply:
solidity 0.8よりoverflow, underflow発生時にrevertされるようになったため考慮不要

Reply:
ref: https://docs.soliditylang.org/en/v0.8.0/080-breaking-changes.html#silent-changes-of-the-semantics

Since we are using v0.8, overflow and underflow in arithmetic operations will be reverted.
> v0.8を使用しているため、算術演算におけるオーバーフローとアンダーフローはrevertされます。